Assets/HoloToolkit-Examples/Sharing/SharingService/Scripts/SyncObjectSpawner.cs

```diff
@@ -33,42 +33,52 @@ private void Awake()
             }
         }
 
-        public void SpawnBasicSyncObject()
+        /// <summary>
+        /// Spawn a sync object at the position and with the rotation you want.
+        /// </summary>
+        public bool SpawnSyncObject(SyncSpawnedObject syncObject, Vector3 position, Quaternion rotation)
         {
-            Vector3 position = Random.onUnitSphere * 2;
-            Quaternion rotation = Random.rotation;
-
-            var spawnedObject = new SyncSpawnedObject();
-
-            spawnManager.Spawn(spawnedObject, position, rotation, spawnParentTransform.gameObject, "SpawnedObject", false);
+            if (!spawnManager.Spawn(syncObject, position, rotation, spawnParentTransform.gameObject, syncObject.Name.Value, false)) {
```

Comment:
why not just return the value from Spawn?

ex: return spawnManager.Spawn( … );